### ISSUE TITLE
How is the title linked to the TextView in the simple example on the readme?

### ISSUE THREAD

**Issue by ataulm** [novoda/landing-strip#18]
Can we update the readme to be more explicit about how the simplest example works?

**Comment by ouchadam** [comment]
because of this 

``` java
    private static final TabSetterUpper SIMPLE_TEXT_TAB_SETTER_UPPER = new TabSetterUpper() {
        @Override
        public View setUp(int position, CharSequence title, View inflatedTab) {
            ((TextView) inflatedTab).setText(title);
            inflatedTab.setFocusable(true);
            return inflatedTab;
        }
    };
```

https://github.com/novoda/landing-strip/blob/master/core/src/main/java/com/novoda/landingstrip/LandingStrip.java#L80

**Comment by ouchadam** [comment]
fixed with #32 by having the api be super explicit about binding the view
